src/test/java/TestClass.java

```diff
@@ -6,48 +6,56 @@
 import org.openqa.selenium.Point;
 
 import java.time.Duration;
-import java.util.Random;
 
 import static com.codeborne.selenide.Selenide.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class TestClass extends BaseTest {
-    public static String baseUrl = "http://automationpractice.com/";
+    static final String BASE_URL = "http://automationpractice.com/";
 
     @Disabled("It is not useful test")
     @Test
-    public void test() {
-        open(baseUrl);
-        $$(".item-img").should(CollectionCondition.size(7)).get(4).scrollTo();
+    public void scrollTo() {
```

Comment:
Example "checkScroll", 
or are you testing the Selenide scrollTo () method?

Reply:
rename everywhere

Reply:
rename